Curve-finance-suite/Unkill/README.md

```diff
@@ -0,0 +1,20 @@
+# UnkillMe Agent
+
+## Description
+
+This agent detects when Curve Pool Owner calls `unkill_me` functions.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- CURVE-12
+  - Fired when Curve Pool Owner calls `unkill_me` function.
+  - Severity is always set to "info".
+  - Type is always set to "info".
+  - Metadata contains:
+    - `UnkillPool`: Contract where `unkill_me` was called.
```

Comment:
If possible, a link to a TX on mainnet that would trigger this agent would be good to prove that it works not just with our test harness, but with real world data too.

Reply:
Sorry, I wasn't able to find any relevant transaction